shakedown/dcos/master.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for working with master"""
+
+from shakedown import *
+
+
+def partition_master(incoming=True, outgoing=True):
+    """ Partition Master's port alone. To keep DC/OS cluster running.
```

Comment:
Some style nits for documentation consistency: lowercase 'M' in "master," and a comma instead of a period after "alone." Could you also document the parameters in Sphinx format? See e.g. `shakedown.dcos.service` for examples.